### PR TITLE
hw-mgmt: scripts: Fix FAN EEPROM insert/removal on SN2201. 		  Common i2c_bus_offset fix.

### PR DESCRIPTION
**Pull request by mshych**
1. Use common helper function for connect/disconenct FAN EEPROMs on SN2201.
2. Fix common usage of i2c_bus_offset. New i2c_bus_offset attribute is added
   to hw-management config directory.

Signed-off-by: Michael Shych <michaelsh@nvidia.com>

### CODE REVIEW

**Comment by mshych** [comment]
RERUN

**Review by felixradensky — APPROVED** [review]

File: usr/usr/bin/hw-management-helpers.sh [hunk 1]
```diff
@@ -182,12 +187,14 @@ change_file_counter()
 
 connect_device()
 {
+	find_i2c_bus
 	if [ -f /sys/bus/i2c/devices/i2c-"$3"/new_device ]; then
 		addr=$(echo "$2" | tail -c +3)
```
Comment (vadimp-nvidia) [inline usr/usr/bin/hw-management-helpers.sh:192]: Why do we need to find addr like addr=$(echo "$2" | tail -c +3)? Before we just pass it as argument.

Comment (mshych) [reply usr/usr/bin/hw-management-helpers.sh:192]: It was added by Sasha in previous commits. I can just say that this working.

**Comment by mshych** [comment]
RERUN

**Review by sholeksandr — APPROVED** [review]

**Pull request merged**